frappe/public/js/frappe/form/controls/link.js

```diff
@@ -327,10 +334,6 @@ frappe.ui.form.ControlLink = class ControlLink extends frappe.ui.form.ControlDat
 						me.$input.cache[doctype][term] = r.message;
 						me.awesomplete.list = me.$input.cache[doctype][term];
 						me.toggle_href(doctype);
-
-						r.message.forEach((item) => {
-							frappe.utils.add_link_title(doctype, item.value, item.label);
-						});
```

Comment:
Is this removed on purpose? What's the rationale?

Reply:
Sorry, it was truly an accidental change, I will fix it right away.